Source/msg.cpp

```diff
@@ -2893,6 +2949,47 @@ void NetSendCmdDelItem(bool bHiPri, uint8_t bLoc)
 		NetSendLoPri(MyPlayerId, (byte *)&cmd, sizeof(cmd));
 }
 
+void NetSendCmdChInvItem(bool bHiPri, int invGridIndex)
+{
+	TCmdChItem cmd;
+
+	int8_t invListIndex = abs(MyPlayer->InvGrid[invGridIndex]) - 1;
+	Item &item = MyPlayer->InvList[invListIndex];
+
+	cmd.bCmd = CMD_CHANGEINVITEMS;
+	cmd.bLoc = invGridIndex;
+	cmd.wIndx = item.IDidx;
+	cmd.wCI = item._iCreateInfo;
+	cmd.dwSeed = item._iSeed;
+	cmd.bId = item._iIdentified ? 1 : 0;
+	cmd.dwBuff = item.dwBuff;

```

Comment:
We have this logic now in three places.
Could we add a helper method `SetItemForTCmdChItem` or something like this?

Reply:
I'm hesitant to do that just yet because I will probably need to replace two of these places with `TCmdPItem` instead.

Reply:
I'm fine with leaving them for now and refactor them later. 🙂